lp-steam/templates/csgo/server.j2

```diff
@@ -22,14 +22,34 @@ sv_lan 1
 host_players_show 2
 host_info_show 2
 
+game_type 0 
+game_mode 1
+
 {% if item.value.gotv | default(false)  %}
+tv_port "{{ item.value.gameport + 5 }}" 
 tv_enable "1"
 tv_title "{{ lp.name }} GOTV {{ steam_server_id }}.{{ item.key }}"
 tv_name "{{ lp.name }} GOTV {{ steam_server_id }}.{{ item.key }}"
 tv_password "{{ csgo_gotv_password | default('stream') }}"
 tv_maxclients "{{ csgo_gotv_maxclients | default(4) }}"
 tv_snapshotrate "{{ csgo_gotv_snaprate | default(64) }}"
-tv_delaymapchange 1
+tv_delaymapchange "1" 
+tv_autorecord "1" 
+tv_allow_camera_man "1" 
+tv_allow_static_shots "1" 
+tv_autoretry "1" 
+tv_chatgroupsize "0" 
+tv_chattimelimit "8" 
+tv_debug "0" 
+tv_delay "120" 
+tv_deltacache "2" 
+tv_dispatchmode "0" 
+tv_maxrate "0" 
+tv_nochat "1" 
```

Comment:
This is a client variable, doesn't exist on server, so won't work as you intended.